src/wp-includes/media.php

```diff
@@ -1914,45 +1922,97 @@ function wp_filter_content_tags( $content, $context = null ) {
 }
 
 /**
- * Adds `loading` attribute to an `img` HTML tag.
+ * Adds optimization attributes to an `img` HTML tag.
  *
- * @since 5.5.0
+ * @since 6.3.0
  *
  * @param string $image   The HTML `img` tag where the attribute should be added.
  * @param string $context Additional context to pass to the filters.
- * @return string Converted `img` tag with `loading` attribute added.
+ * @return string Converted `img` tag with optimization attributes added.
  */
-function wp_img_tag_add_loading_attr( $image, $context ) {
-	// Get loading attribute value to use. This must occur before the conditional check below so that even images that
-	// are ineligible for being lazy-loaded are considered.
-	$value = wp_get_loading_attr_default( $context );
-
-	// Images should have source and dimension attributes for the `loading` attribute to be added.
+function wp_img_tag_add_loading_optimization_attrs( $image, $context ) {
+	// Images should have source and dimension attributes for the loading attributes to be added.
```

Comment:
Nit-pick, to avoid confusion with the actual `loading` attribute.
```suggestion
	// Images should have source and dimension attributes for the loading optimization attributes to be added.
```

Reply:
Addressed as part of https://github.com/WordPress/wordpress-develop/pull/4495/commits/67a199b06fc378e541b758c7fa06313a6b6cec9c